src/go-postgres/devcontainer-template.json

```diff
@@ -7,19 +7,18 @@
     "publisher": "Dev Container Spec Maintainers",
     "licenseURL": "https://github.com/devcontainers/templates/blob/main/LICENSE",
     "options": {
-		"imageVariant": {
+        "imageVariant": {
             "type": "string",
             "description": "Go version (use -bookworm, or -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
                 "1-bookworm",
+                "1.21-bookworm",
```

Comment:
Unfortunately, this is in progress. We need to wait for https://github.com/devcontainers/images/pull/701 to be merged in and the images need to be released for this variant.